mail_tracking/models/mail_tracking_event.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+# © 2016 Antonio Espinosa - <antonio.espinosa@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import time
+from datetime import datetime
+
+from openerp import models, api, fields
+import openerp.addons.decimal_precision as dp
+
+
+class MailTrackingEvent(models.Model):
+    _name = "mail.tracking.event"
+    _order = 'timestamp desc'
+    _rec_name = 'event_type'
+    _description = 'MailTracking event'
+
+    recipient = fields.Char(string="Recipient", readonly=True)
+    timestamp = fields.Float(
+        string='UTC timestamp', readonly=True,
+        digits=dp.get_precision('MailTracking Timestamp'))
+    time = fields.Datetime(string="Time", readonly=True)
+    date = fields.Date(string="Date", readonly=True)
```

Comment:
The date field is use to group messages by month in the search view. time field is used as a column in the tree view.